### PR TITLE
bump ansible community.kubernetes depedency to <1.1.0

### PR DESCRIPTION
**Pull request by jberkhahn**
Not sure if this will work.

### CODE REVIEW

File: internal/plugins/ansible/v1/scaffolds/internal/templates/requirements.go [hunk 1]
```diff
@@ -34,6 +34,6 @@ func (f *RequirementsYml) SetTemplateDefaults() error {
 const requirementsYmlTmpl = `---
 collections:
   - name: community.kubernetes
-    version: "<1.0.0"
+    version: "<1.1.0"
```
Comment (estroz) [inline internal/plugins/ansible/v1/scaffolds/internal/templates/requirements.go:37]: Perhaps we can do this so the latest patch release is used. I'm not sure what compatibility looks like for minor versions, so this is a reasonable compromise.
```suggestion
    version: "~=1.1"
```

Comment (jberkhahn) [reply internal/plugins/ansible/v1/scaffolds/internal/templates/requirements.go:37]: im just testing stuff. i'm expecting stuff to explode from <1.0.0 to 1.0.0

Comment (jberkhahn) [reply internal/plugins/ansible/v1/scaffolds/internal/templates/requirements.go:37]: but yes in the final version i think i'll do something like that

Comment (estroz) [reply internal/plugins/ansible/v1/scaffolds/internal/templates/requirements.go]: Supposedly the [`~=` operator](https://www.python.org/dev/peps/pep-0440/#compatible-release) should work, but I guess not.

**Comment by jmrodri** [comment]
@fabianvf mentioned to me "I think that will pin it to 1.x, which we probably don't want to do until the k8s_status collection is fixed" so hold this PR for now

**Comment by camilamacedo86** [comment]
/hold

Until make the 1.1.0 works with the projects

**Comment by theishshah** [comment]
I believe we need the '>=1.1.0' operator. Once we have this we can replicate the dependency update downstream

**Comment by camilamacedo86** [comment]
Hi @jberkhahn, 

This pr requires to be rebased with the master. It is in a very old stage.
I think we might already in the stage to make the operator works with the upper versions.
Maybe the problem faced with 1.1.0 is solved. 

Could you please rebase it with teh master and check if we are in the place to in fact use the upper version?
Otherwise we can track an issue for that (if do not exist) and close this one. WDYT?

**Comment by estroz** [comment]
This is being handled by #4594 now.

**Pull request closed without merging**